src/fetch.rs

```diff
@@ -0,0 +1,56 @@
+/// Fetches the latest version of Amethyst by pulling from crates.io
+/// Most of this code is based off of cargo-edit's fetch code
+use reqwest;
+use semver;
+use serde_json as json;
+use std::time::Duration;
+use env_proxy;
```

Comment:
Imports should be ordered as follows:

1. std imports
2. imports from other crates
3. local imports

with a newline between after each section. Also, please sort them alphabetically (this should be done by rustfmt if there's a config already).